### ISSUE TITLE
[Bug] Options doesn't work in 2.4.0 version

### ISSUE THREAD

**Issue by Kristinita** [naokazuterada/MarkdownTOC#100]
### 1. Summary

All MarkdownTOC options doesn't work for me in 2.4.0 version. In previous versions all options works good.

### 2. Settings

**Operating system and version:**
Windows 10 Enterprise LTSB 64-bit EN
**Sublime Text:**
Build 3126
**Package:**
The latest stable version of MarkdownTOC for Sublime Text 3

My `MarkdownTOC.sublime-settings` file:

```json
{
    "default_autoanchor": true,
    "default_autolink": true,
    "default_bracket": "round",
    "default_depth": 4,
    "default_style": "ordered",
    "default_uri_encoding": false
}

```

My `TestMarkdownTOC.md` file:

```markdown
# SashaFirstHeader

## SashaSubheader

# SashaSecondHeader

```

### 3. Steps to reproduce

I reproduce the problem in a version of Sublime Text without plugins and user settings.

I place my carriage in start of `TestMarkdownTOC.md` file → <kbd>Ctrl+Shift+P</kbd> (<kbd>⌘⇧p</kbd> for Mac) → `MarkdownTOC: Insert TOC`.

### 4. Expected behavior

If I use [**one of previous version**](https://github.com/naokazuterada/MarkdownTOC/commit/5b57b3b54d4984f7059e2a887c5e186004af5036):

```markdown
<!-- MarkdownTOC -->

1. [**SashaFirstHeader**](#sashafirstheader)
    1. [**SashaSubheader**](#sashasubheader)
1. [**SashaSecondHeader**](#sashasecondheader)

<!-- /MarkdownTOC -->


<a name="sashafirstheader"></a>
# SashaFirstHeader

<a name="sashasubheader"></a>
## SashaSubheader

<a name="sashasecondheader"></a>
# SashaSecondHeader
```

### 5. Actual behavior

If 2.4.0 version:

```markdown
<!-- MarkdownTOC -->

- [SashaFirstHeader]
    - [SashaSubheader]
- [SashaSecondHeader]

<!-- /MarkdownTOC -->

# SashaFirstHeader

## SashaSubheader

# SashaSecondHeader
```

Thanks.

**Comment by naokazuterada** [comment]
@Kristinita 
Thank you for your reporting, and sorry...
I will look at this as soon as possible.

**Comment by xakraz** [comment]
Same issue for me.
Starting today 2017-08-03, 
* my package has been updated to version `"version": "2017.08.02.01.05.25"`.
* All the Toc links are not generated

Using the console I see that the USER Preference file is not loaded.

```
reloading plugin MarkdownTOC.MarkdownTOC
reloading settings Packages/MarkdownTOC/MarkdownTOC.sublime-settings
reloading settings Packages/User/Package Control.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings
```

The file `Packages/User/MarkdownTOC.sublime-settings` is ignored

**Comment by naokazuterada** [comment]
Oh, so sorry.

Something goes wrong with 2.4.0...
I want to fix it, but I have no time these days. So [I just revert the master](83b2555ca15fc0a62fa4785dd96829e516468f73) (now it's same as 2.3.2).

Anyway thank you for your reporting. And I will fix it ASAP. 🤕

**Comment by naokazuterada** [comment]
I'm so sorry for my late dealt, but this problem was fixed in [2.4.1](https://github.com/naokazuterada/MarkdownTOC/releases/tag/2.4.1).
Please check it, and reopen this issue if you have any problems yet.
